operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/KubernetesDependentResource.java

```diff
@@ -136,9 +143,17 @@ protected NonNamespaceOperation<R, KubernetesResourceList<R>, Resource<R>> prepa
 
   @Override
   protected InformerEventSource<R, P> createEventSource(EventSourceContext<P> context) {
-    configureWith(null, context.getControllerConfiguration().getNamespaces(), true);
-    log.warn("Using default configuration for " + resourceType().getSimpleName()
-        + " KubernetesDependentResource, call configureWith to provide configuration");
+    if (kubernetesDependentResourceConfig != null) {
```

Comment:
@metacosm it's done here, when the event source is created.

Reply:
🤔 I saw that but wasn't convinced there wasn't a path where this wouldn't get called… But I guess `createEventSource` is always called so we should be good! Thanks!